### PR TITLE
Galaxy settings editable

### PR DESCRIPTION
**Pull request by o01eg**
This PR is move out from https://github.com/freeorion/freeorion/pull/1660

It introduce new option for the host to delegate rights to manage galaxy and AI settings.

### CODE REVIEW

**Comment by geoffthemedio** [comment]
The layout of the multiplayer setup screen is a bit off with the extra option. Needs a bit more space.

Also, put the button at the bottom of the options, not the top.

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -1941,6 +1941,9 @@ Previous Player
 MULTIPLAYER_PLAYER_LIST_STARTING_SPECIES
 Starting Species
 
+EDITABLE_GALAXY_SETTINGS
+Anyone can edit galaxy and AI settings.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: This name is a bit long for the checkbox itself. Maybe "All Can Edit Settings" for the label, and then add an (even more so than this) longer tooltip explaining what the option does.

**Comment by o01eg** [comment]
Rebased. Moved button bottom and added tooltip. But somehow the tooltip doesn't shown.

File: UI/MultiplayerLobbyWnd.cpp [hunk 2]
```diff
@@ -751,20 +758,24 @@ void MultiPlayerLobbyWnd::LobbyUpdate(const MultiplayerLobbyData& lobby_data) {
...
-    if (send_update_back && ThisClientIsHost())
+    if (send_update_back && (ThisClientIsHost() || m_lobby_data.m_any_can_edit))
         SendUpdate();
 
     LogPlayerSetupData(m_lobby_data.m_players);
 }
 
 void MultiPlayerLobbyWnd::Refresh() {
-    if (ThisClientIsHost()) {
+    m_any_can_edit->Disable(! ThisClientIsHost());
```
Comment (geoffthemedio) [inline UI/MultiplayerLobbyWnd.cpp]: remove space after `!`

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -1941,6 +1941,12 @@ Previous Player
 MULTIPLAYER_PLAYER_LIST_STARTING_SPECIES
 Starting Species
 
+EDITABLE_GALAXY_SETTINGS
+All Can Edit Settings
+
+EDITABLE_GALAXY_SETTINGS_DESC
+Anyone can edit galaxy and AI settings.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Anyone -> All players

Comment (o01eg) [reply default/stringtables/en.txt]: Fixed

File: UI/MultiplayerLobbyWnd.cpp (hunk 2 above)
Comment (o01eg) [reply UI/MultiplayerLobbyWnd.cpp]: Fixed.

**Comment by o01eg** [comment]
Add additional padding between the file button and the option's check box.

**Pull request merged**